### PR TITLE
Added issue & PR templates, closes #151

### LINKED ISSUE #151: Create issue & PR templates
https://github.com/blog/2111-issue-and-pull-request-templates

Also do this for the other repos

### PR DESCRIPTION
**Pull request by divayprakash**

### CODE REVIEW

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+* Provide a general summary of the issue in the **Title** above.  
+* Before you open an issue, please check if a similar issue already exists or has been closed before.
+
+### Important
+Mark with [x] to select. Leave as [ ] to unselect.
+
+### When reporting a bug/issue, please be sure to include the following:
+- [ ] Screenshot
+- [ ] The browser, browser version and the opearting system you're using
```
Comment (shakeelmohamed) [inline ISSUE_TEMPLATE.md]: typo here, should be `operating`

File: ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+- [ ] The behavior you expect to see, and the actual behavior
+- [ ] Steps to reproduce the behavior 
+- [ ] Possible solution - _**not**_ obligatory, but suggest a fix/reason for the bug
```
Comment (shakeelmohamed) [inline ISSUE_TEMPLATE.md]: Instead of this can we just say `(optional) Possible solution/fix/workaround`?

File: ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+### When you open an issue for a change/improvement/feature request, please add as much detail as possible:
+- [ ] A description of the problem you're trying to solve, including _why_ you think this is a problem
+- [ ] If the feature changes current behavior, reasons why your solution is better
+- [ ] Possible solution - _**not**_ obligatory, but suggest ideas how to implement the change/improvement/feature
```
Comment (shakeelmohamed) [inline ISSUE_TEMPLATE.md]: Same comment as above for `(optional)...`

File: ISSUE_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+* Provide a general summary of the issue in the **Title** above.  
+* Before you open an issue, please check if a similar issue already exists or has been closed before.
+
+### Important
+Mark with [x] to select. Leave as [ ] to unselect.
+
+### When reporting a bug/issue, please be sure to include the following:
```
Comment (shakeelmohamed) [inline ISSUE_TEMPLATE.md]: This heading is pretty verbose, how about just `### When reporting a bug/issue`?

File: ISSUE_TEMPLATE.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+- [ ] Screenshot
+- [ ] The browser, browser version and the opearting system you're using
+- [ ] The behavior you expect to see, and the actual behavior
+- [ ] Steps to reproduce the behavior 
+- [ ] Possible solution - _**not**_ obligatory, but suggest a fix/reason for the bug
+
+### When you open an issue for a change/improvement/feature request, please add as much detail as possible:
```
Comment (shakeelmohamed) [inline ISSUE_TEMPLATE.md]: This heading is pretty verbose, how about just `### When you open an issue for a change/improvement/feature request`?

File: PULL_REQUEST_TEMPLATE.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+### Description
+- [ ] Describe your changes in detail.
+
+### Final checklist:
+Go over all the following points and check all the boxes that apply  
+If you're unsure about any of these, don't hesitate to ask. We're here to help!
+- [ ] My code follows the code style of this project.
+- [ ] My change requires a change to the documentation.
+- [ ] I have updated the documentation accordingly.
+- [ ] I have read the **CONTRIBUTING** guidelines.
```
Comment (shakeelmohamed) [inline PULL_REQUEST_TEMPLATE.md]: Let's put a link to the contributing notes

**Comment by shakeelmohamed** [comment]
Thanks for doing this @divayprakash!

Can we also move these files, and the contributing notes into a `.github` folder? This feature was added at the same time, quoted below:


> If you're worried about the added clutter in the root directory of your project, we also added support for a .github/ folder. You can put CONTRIBUTING.md, ISSUE_TEMPLATE.md, and PULL_REQUEST_TEMPLATE.md files in .github/ and everything will work as expected.

**Comment by divayprakash** [comment]
Working on it now @shakeelmohamed

**Comment by divayprakash** [comment]
@shakeelmohamed Have a look, its's all done!

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+### Description
+- [ ] Describe your changes in detail.
+
+### Final checklist:
+Go over all the following points and check all the boxes that apply  
+If you're unsure about any of these, don't hesitate to ask. We're here to help!
+- [ ] My code follows the code style of this project.
+- [ ] My change requires a change to the documentation.
+- [ ] I have updated the documentation accordingly.
+- [ ] I have read the [**CONTRIBUTING**](https://github.com/zen-audio-player/zen-audio-player.github.io/blob/master/CONTRIBUTING.md) guidelines.
```
Comment (shakeelmohamed) [inline .github/PULL_REQUEST_TEMPLATE.md]: The URL can just be `.github/CONTRIBUTING.md`.

**Comment by shakeelmohamed** [comment]
One other thing, is updating the link to the contributing notes in the README

**Comment by divayprakash** [comment]
on it

**Comment by divayprakash** [comment]
@shakeelmohamed

**Comment by shakeelmohamed** [comment]
Thanks, welcome to the org!

**Comment by divayprakash** [comment]
You're welcome! :+1:

**Pull request merged**